### DISCUSSION TITLE
Resource timeline possibility

### DISCUSSION THREAD

**Discussion by komitoff** [vkurko/calendar#217] [Q&A]
Hello there, 
Great work with the component you design. Since it looks very similar to the fullcalendar.io I was wondering is it possible to achieve something similar to what they have which is resource timeline(like in the image bellow). 
![image](https://github.com/vkurko/calendar/assets/7748256/d03c20a1-fdb4-4117-b4ac-9e96f850207a)
Thanks in advance!

**Comment by vkurko** [comment]
Hello,

Thank you for your interest in Event Calendar. Regarding the timeline view please refer to this [issue](https://github.com/vkurko/calendar/issues/30).
